script.js

```diff
@@ -1,406 +1,292 @@
-/*Here we have created two different arrays that you can work with if you want.
-If you choose to create your own arrays with elements, just make sure that some
-of the properties make sense to filter on, and some to sort on.*/
+//Dom selectors
+const destinationDiv = document.getElementById('load-destinations')
+const buttonTemp = document.getElementById('button-temp')
+const buttonRate = document.getElementById('button-rate')
+const dropDown = document.getElementById('filter-dropdown')
+const buttonRandom = document.getElementById('button-random')
+const searchInput = document.getElementById("search")
 
-const books = [
+//global scope
+let lowToHigh = true
+
+//Array with objects with different destinations
+const destinations = [
```

Comment:
Really like that you made you're own array of objects! Such a fun touch. 
/Martin & Mai